scripts/send_application_on_closure.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+from datetime import datetime
+
+sys.path.insert(1, ".")
+
+import api.routes.application.helpers as helpers
+from app import app
+from config import Config
+from db.models.applications import ApplicationsMethods
+from db.models.forms import FormsMethods
+from external_services.models.notification import Notification
+from flask import current_app
+
+
+def send_email_on_deadline_task(fund_id, round_id):
```

Comment:
Can we call this `send_incomplete_applications_after_deadline` or something more descriptive?

Reply:
ahh yes, have changed so it's more descriptive now :)